### PR TITLE
feat(stats-detector-spanOnbreakdown): Updated design.

### PR DESCRIPTION
**Pull request by Abdkhan14**
For issue: https://github.com/getsentry/sentry/issues/56015

Updated the spanOpbreakdown designs: 

Before: 
Screenshot 2023-10-20 at 10 30 04 AM

After: 
Screenshot 2023-10-20 at 1 37 35 PM

### CODE REVIEW

File: static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx [hunk 1]
```diff
@@ -95,106 +104,190 @@ class PieChart extends Component<Props> {
...
+                    skipWrapper
+                    title={t(
+                      `Total time for %s went %s from %s to %s`,
+                      op,
+                      percentageText,
+                      oldValue,
+                      newValue
+                    )}
+                  >
+                    <SpanOpChange regressed>
```
Comment (narsaynorath) [inline static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx:166]:
```suggestion
                    <SpanOpChange regressed={change > 0}>
```

File: static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx [hunk 2]
```diff
@@ -95,106 +104,190 @@ class PieChart extends Component<Props> {
...
+              avoidLabelOverlap: false,
               label: {
+                position: 'inside',
+                formatter: params => {
+                  return `${Math.round(Number(params.percent))}%`;
+                },
                 show: true,
+                color: theme.background,
+                fontSize: 12,
+                fontWeight: 600,
```
Comment (narsaynorath) [inline static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx:237]:
```suggestion
                formatter: params => {
                  return `${params.name} ${Math.round(Number(params.percent))}%`;
                },
                show: true,
                color: theme.background,
                overflow: 'break',
                width: 40
```

Screenshot 2023-10-20 at 4 34 20 PM

I don't think the font-weight is adding too much here, it looks crisper without it. These other settings let us show the param name and break it up into two lines too. It's a bit of a hack but it makes the segments clearer without the hovering

**Review by narsaynorath — APPROVED** [review]
Some small comments but looks good to me overall. Thanks for the upgrade!

File: static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx (hunk 1 above)
Comment (Abdkhan14) [reply static/app/components/events/eventStatisticalDetector/aggregateSpanOps/pieChart.tsx:166]: Woops good catch @narsaynorath

**Pull request merged**